CONTRIBUTING.md

```diff
@@ -5,8 +5,25 @@ Thank you for your interest in contributing to Rocket Chip!
 
 ## Table of Contents
 
++ [Submitting a PR](#submitting)
++ [Conditions for Merging a PR](#merging)
 + [Bumping Submodules](#bumping)
 
+### <a name="merging"></a> Submitting a PR
```

Comment:
is the `name` here supposed to be `"submitting"`?

Reply:
yep. fixing